mlrun/projects/project.py

```diff
@@ -2081,20 +2073,21 @@ def update_model_monitoring_controller(
             image=image,
         )
 
-    def disable_model_monitoring(self):
+    def disable_model_monitoring(
+        self, *, delete_histogram_data_drift_app: bool = True
+    ) -> None:
+        """
+        Disable model monitoring by deleting the underlying functions infrastructure from MLRun database.
+
+        :param delete_histogram_data_drift_app: Whether to delete the histogram data drift app.
+        """
```

Comment:
In the end of 1.7.0 this function will delete also the nuclio pods of those function, and without this ability this function is more harm than good, therefore I think for now lets delete the  mention it in the docs and maybe we can write also that this function is not valid for now.